core/src/main/java/curso/modelo/level.java

```diff
@@ -0,0 +1,5 @@
+package curso.modelo;
+
+public enum level {
```

Comment:
Poner en mayúscula inicial
```suggestion
public enum Level {
```